index.html

```diff
@@ -123,92 +118,13 @@ <h3 id="howcanihelp">How can I help?</h3>
         <p>
           If you have a compatible Chromebook, you can fork my Github repo and add new packages to the <code>packages/</code> directory if you managed to build them from source successfully on your device. Package recipes are simple Ruby files - here is an example:
         </p>
-        <pre># It is recommended that you investigate other files in the 'packages/' directory
-# to help get an idea of how existing packages are being installed through Chromebrew.
-
-require 'package'
-
-class Template &lt; Package
-  description ''
-  homepage ''
-  version ''
-  license '' # Package license
-  compatibility '' # Package compatibility
-  source_url '*.tar.{gz,bz2,lz,etc}'
-  source_sha256 ''
-
-# Optional: Prebuilt binary
-#  binary_url ({
-#    aarch64: '',
-#     armv7l: '',
-#       i686: '',
-#     x86_64: '',
-#  })
-#  binary_sha256 ({
-#    aarch64: '',
-#     armv7l: '',
-#       i686: '',
-#     x86_64: '',
-#  })
-
-
-  # Register dependencies (use the following line as a basis)
-  #
-  # depends_on '*'
-  #
-
-  # Function for checks to see if install should occur.
-  def self.preflight
-
-  end
-
-  # Function to perform pre-install operations prior to install for both source build and binary distribution.
-  def self.preinstall
-
-  end
-
-
-  # Function to perform patch operations prior to build from source.
-  def self.patch
-
-  end
-
-
-  # Function to perform build from source.
-  def self.build
-
-  end
-
-  # Function to perform pre-install operations prior to install.
-  def self.preinstall
-
-  end
-
-  # Function to perform install from source build.
-  def self.install
-
-  end
-
-
-  # Function to perform post-install for both source build and binary distribution
-  def self.postinstall
-
-  end
-
-
-  # Function to perform check from source build.
-  # This is execute if and only if `crew build`.
-  def self.check
-
-  end
-
+        <script id="embed_template"></script>
+        <script src="js/embed_template.js"></script>
```

Comment:
Move to `<head>` section with `defer`?  Just curious if it will work.

Reply:
It works